mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -365,3 +365,20 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6132 applyArmorToDamage (Lnet/minecraft/class_1282;F)F
 		ARG 1 source
 		ARG 2 amount
+	METHOD method_23883 dropXp ()V
+	METHOD method_23733 onKilledBy (Lnet/minecraft/class_1309;)V
+		COMMENT Performs secondary effects after this mob has been killed.
+		COMMENT 
+		COMMENT <p> The default behaviour spawns a wither rose if {@code adversary} is a {@code WitherEntity}.
+		COMMENT 
+		COMMENT @param adversary the main adversary responsible for this entity's death
+		ARG 1 adversary
+	METHOD method_23329 computeFallDamage (FF)I
+		ARG 1 fallDistance
+		ARG 2 damageMultiplier
+	METHOD method_23328 playBlockFallSound ()V
+	METHOD method_21754 isHoldingLadder ()Z
```

Comment:
This name sounds to me as if the entity is holding a ladder item. I'd say `isHeldInLadder` is more clear (you can come up with other better names either)

Reply:
`isHoldingOntoLadder` ?